### DISCUSSION TITLE
Comparing SCENIC results between samples

### DISCUSSION THREAD

**Discussion by ocqub** [aertslab/SCENIC#242] [* General SCENIC questions]
Hi,
Are there any recommendations for comparing SCENIC results between samples (e.g. two single-cell datasets, run through SCENIC separately), other than comparing the heatmaps and clustering/tSNE? 

For instance, would it be appropriate to export the continuous/binary SCENIC matrices from two samples into another snRNA-seq analysis package and perform differential expression analysis, using the regulon scores in lieu of 'real' gene expression data? I'm hoping this would provide insight into which regulons are changing and to what extent, but am unsure if it's as simple as comparing scores in this way.

Thanks for any suggestions.

**Comment by saeedfc** [comment]
Hi,
@ocqub  and @s-aibar
Just would like to add a thread to the question.

I use normalized counts from processed seurat object(Which is merged between samples using seurat). 
So,
If I derive AUCELL matrix for an expression matrix of n cells, Can I still use the same AUCELL matrix to work with a subset of the data (say m cells, m<n) ? Or do I have to rerun SCENIC for the subset of m cells of the data separately? (Asking because I don't know whether there is any pan data nomalization or dependency generated internally in the workflow. I am using GRNBoost2 from pyscenic)
